catalog/ui/search-ui/simple/src/main/resources/SearchPage.jsp

```diff
@@ -475,5 +475,29 @@
 
  	<script type="text/javascript" src="js/Search-min.js?bust=${timestamp}"></script>
 
+    <!-- Usage Modal -->
+    <!-- Only show if the modal is enabled -->
+    <% if(props.getSystemUsageEnabled() == true) { %>
+    <!-- Do not allow users to click outside of the modal to dismiss it, the OK button must be pressed -->
+    <script>
+        jQuery(function () {
+            $('#usageModal').modal({
+                backdrop: 'static',
+                keyboard: false
+            })
+        })
+    </script>
+    <div id="usageModal" class="modal hide" tabindex="-1" role="dialog"
+        aria-labelledby="myModalLabel" aria-hidden="true">
+        <div class="modal-header">
+            <h3><%=props.getSystemUsageTitle()%></h3>
```

Comment:
❗ Unless we're already sanitizing it somewhere else, we should make sure to escape html anywhere that user/admin input is being displayed. I think JSP has some built in methods to escape html characters or they can probably be escaped within the get methods themselves.

Reply:
Great catch, @garrettfreibott ! I'll look into what JSP has as far as escaping stuff like this otherwise I'll see what I can do at the Java level.